src/flinkStatementResultsManager.ts

```diff
@@ -329,9 +372,6 @@ export class FlinkStatementResultsManager {
         this._filteredResults(this.filterResultsBySearch());
         return null;
       }
-      case "GetSearchQuery": {
```

Comment:
This a GetMaxSize were unused leftovers from consume.ts cloning I guess?

Reply:
Yes and no -- `GetSearchQuery` is unused from cloning. `GetMaxSize` was lingering remnant from when I removed the "max results" dropdown.